addons/mail/static/src/core/common/thread_model.js

```diff
@@ -463,6 +463,37 @@ export class Thread extends Record {
         },
     });
 
+    newMessageSeparatorOwner = Record.one("Message", {
+        compute() {
+            if (!this.selfMember) {
+                return false;
+            }
+            const messages = this.nonEmptyMessages;
+            const separator = this.selfMember.localNewMessageSeparator;
```

Comment:
I don't think "owner" is a good choice for naming this field: this is too obscure what is meant by "owning" the new message separator in this context.

What about something like `messageAfterNewMessageSeparator`?

Reply:
I had the same thought, to ease the understanding I renamed this field `firstUnreadMessage`. Does it work for you?

Reply:
LGTM, thanks!